packages/lib-react-components/src/CloseButton/CloseButton.spec.js

```diff
@@ -1,33 +1,23 @@
-import { shallow } from 'enzyme'
-import sinon from 'sinon'
-import CloseButton from './CloseButton'
-import { expect } from 'chai'
+import { render, screen } from '@testing-library/react'
+import { composeStory } from '@storybook/react'
+import Meta, { Default, Light, Disabled } from './CloseButton.stories.js'
 
-describe('<CloseButton />', function () {
-  let wrapper
-  before(function () {
-    wrapper = shallow(<CloseButton closeFn={sinon.spy()} />)
+describe('Component > CloseButton', function () {
+  it('renders the default button', function () {
+    const DefaultStory = composeStory(Default, Meta)
+    render(<DefaultStory />)
+    expect(screen.getByRole('button', { disabled: false })).to.be.ok()
```

Comment:
⚠️ Ahhah, we've got a small problem here - `disabled: false/true` isn't actually a proper option for getRole()

Check out the `options?` listed in [RTL's ByRole documentation](https://testing-library.com/docs/queries/byrole) -
there's `hidden`, `name`, `description`, etc, but you can't select by disabled state.

You can test this yourself by changing `disabled: false` to `disabled: true`, and the `expect()` will _still_ say, yep, that's definitely `ok()`. This means that getByRole() is ignoring the invalid "disabled" option.

Reply:
One option is to do check the `disabled` attribute separately.

For example, we can replace...
```
expect(screen.getByRole('button', { disabled: false })).to.be.ok()
```

With...
```
const button = screen.getByRole('button')
expect(button).to.be.ok()
expect(button.hasAttribute('disabled')).to.be.false()
```